charts/security-apps/README.md

```diff
@@ -43,6 +43,13 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | gangway.repoURL | string | [repo](https://charts.helm.sh/stable/) | Repo URL |
 | gangway.targetRevision | string | `"0.4.*"` | [gangway Helm chart](https://github.com/helm/charts/tree/master/stable/gangway/) version |
 | gangway.values | object | [upstream values](https://github.com/helm/charts/tree/master/stable/gangway/values.yaml) | Helm values |
+| opa | object | - | [Open Policy Agent](https://www.openpolicyagent.org/) ([example](./examples/opa.yaml)) |
+| opa.chart | string | `"opa"` | Chart |
+| opa.destination.namespace | string | `"infra-opa"` | Namespace |
+| opa.enabled | bool | `false` | Enable Open Policy Agent |
+| opa.repoURL | string | [repo](https://charts.helm.sh/stable) | Repo URL |
```

Comment:
Is there really no official chart in https://github.com/open-policy-agent ?

Or can not this be done with gatekeeper?